hash/scv.xml

```diff
@@ -355,25 +356,26 @@ Information found at http://www.rhod.fr/yeno_epoch.html
 		<year>1986</year>
 		<publisher>Epoch</publisher>
 		<part name="cart" interface="scv_cart">
-			<feature name="slot" value="rom64k" />
-			<dataarea name="rom" size="65536">
-				<rom name="skykid.bin" size="65536" crc="f49f2085" sha1="7342fe10d38c79ed12fbfa94b5fd4bf1266c2571" offset="0" />
+			<feature name="slot" value="rom64k"/>
+			<dataarea name="rom" size="0x10000">
+				<rom name="skykid.bin" size="0x10000" crc="f49f2085" sha1="7342fe10d38c79ed12fbfa94b5fd4bf1266c2571"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="starspdr" supported="partial">
+	<software name="starspdr" supported="no">
 		<description>Star Speeder</description>
 		<year>1985</year>
 		<publisher>Epoch</publisher>
+		<notes>Game does not start.</notes>
 		<part name="cart" interface="scv_cart">
-			<feature name="slot" value="rom64k" />
-			<dataarea name="rom" size="65536">
-				<rom name="starspeeder.1" size="8192"  crc="7978c4a6" sha1="6bb5ff3946867fa7adb6561dc084ef5a7a92c507" offset="0" />
-				<rom size="8192" offset="0x2000" loadflag="reload" />
-				<rom size="8192" offset="0x4000" loadflag="reload" />
-				<rom size="8192" offset="0x6000" loadflag="reload" />
-				<rom name="starspeeder.0" size="32768" crc="511576ee" sha1="1b61132af26dc8d5c1b5e0f72c2e96192bba7e20" offset="0x8000" />
+			<feature name="slot" value="rom64k"/>
+			<dataarea name="rom" size="0x10000">
+				<rom name="starspeeder.1" size="0x2000" crc="7978c4a6" sha1="6bb5ff3946867fa7adb6561dc084ef5a7a92c507"/>
+				<rom size="0x2000" offset="0x2000" loadflag="reload"/>
+				<rom size="0x2000" offset="0x4000" loadflag="reload"/>
+				<rom size="0x2000" offset="0x6000" loadflag="reload"/>
+				<rom name="starspeeder.0" size="0x8000" crc="511576ee" sha1="1b61132af26dc8d5c1b5e0f72c2e96192bba7e20" offset="0x8000"/>
```

Comment:
I’m not really a fan of having implicit offset for some ROMs and explicit offsets for others, particularly within a single `part` like this.